### DISCUSSION TITLE
Welcome to Easy-Web-TV-M3u8 Discussions!

### DISCUSSION THREAD

**Discussion by zhangboheng** [zhangboheng/Easy-Web-TV-M3u8#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajmi982** [comment]
Hi, using CHROME... cant get this to work. Only see this error "The media could not be loaded, either because the server or network failed or because the format is not supported." thanks

**Reply by zhangboheng** [reply]
I see, maybe it's the video source can't work
